Send vacuum to specific area (#44083)
home-assistant/home-assistant.github.io: source/_integrations/vacuum.markdown

## Creating an automation to send the vacuum to clean specific areas

When your vacuum supports area cleaning, you can create an automation in Home Assistant to send your vacuum to clean specific areas. For example, you can set up an automation to have the vacuum clean the living room every day at 2 PM.

Creating such an automation involves two steps:

1. [Mapping your vacuum areas to Home Assistant areas](#mapping-your-vacuum-areas-to-home-assistant-areas). This is a one-time setup step to link the areas defined in your vacuum's app to the areas defined in Home Assistant.
2. [Sending your vacuum to clean specific areas](#sending-your-vacuum-to-clean-specific-areas). This is where you create the automation that tells your vacuum to clean specific areas based on certain triggers.

### Mapping your vacuum areas to Home Assistant areas

Before you can send your vacuum to clean specific areas, you need to map the areas of your vacuum to the areas in Home Assistant.

1. Go to {% my entities title="**Settings** > **Devices & services** > **Entities**" %} and select your vacuum entity.
2. In the entity settings, select the cogwheel {% icon "mdi:cog-outline" %}.

> 3. Select **Map vacuum to areas**.

Reviewer: ```suggestion
3. Select **Map vacuum segments to areas**.
```
This is how it's called technically in the UI right now.

We could streamline it the way you wrote it here, but that then should be changed in the frontend first.

Author: thx for the catch!

   - Result: In the dialog, you should see the vacuum areas listed on the left and the Home Assistant areas on the right.

>    - If you do not see the **Map vacuum to areas** option, your vacuum does not support area cleaning. You cannot use the `vacuum.clean_area` action with this vacuum.

Author: ```suggestion
   - If you do not see the **Map vacuum segments to areas** option, your vacuum does not support area cleaning. You cannot use the `vacuum.clean_area` action with this vacuum.
```

>    - Troubleshooting: If there are no areas listed on the left hand side, make sure the areas are properly set up in the vacuum's app. You might need to save or confirm them again in the app.

Author: ```suggestion
   - Troubleshooting: If there are no areas listed on the left, make sure the areas are properly set up in the vacuum's app. You might need to save or confirm them again in the app.
```

> 4. In the **Map vacuum to areas** dialog, map the Home Assistant areas to the listed areas of your vacuum.

Author: ```suggestion
4. In the **Map vacuum to areas** dialog, for each area reported by your vacuum, select the corresponding Home Assistant area.
```

> 4. In the **Map vacuum to areas** dialog, for each area reported by your vacuum, select the corresponding Home Assistant area.

Author: ```suggestion
4. In the **Map vacuum segments to areas** dialog, for each area reported by your vacuum, select the corresponding Home Assistant area.
```
